src/App.tsx

```diff
@@ -1,61 +1,121 @@
+import { useState } from 'react';
 import './App.scss';
-
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+import { Todo, User, TodosWithUser } from './types/Types';
 
 export const App = () => {
+  const users: User[] = usersFromServer;
+  const todos: Todo[] = todosFromServer;
+
+  function preparedTodosWithUsers(): TodosWithUser[] {
+    return (
+      todos.map(todo => ({
+        ...todo,
+        user: users.find(user => user.id === todo.userId) || null,
+      }))
+    );
+  }
+
+  const [selectedUser, setSelectedUser] = useState('');
+  const [currentTitle, setCurrentTitle] = useState('');
+  // eslint-disable-next-line max-len
+  const [todosWithUsers, setTodosWithUsers] = useState<TodosWithUser[]>(preparedTodosWithUsers());
```

Comment:
Just wrap instead use eslint disable
```suggestion
  const [
   todosWithUsers,
   setTodosWithUsers
 ] = useState<TodosWithUser[]>(preparedTodosWithUsers());
```